### PR TITLE
Finish .NET New IA conversion

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/finish-dotnet-new-ia/sdk/dotnet/

Still need to finish "modify an object schema" but other than that most things are in place.

### CODE REVIEW

File: source/sdk/android/fundamentals/relationships.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-.. _android-client-relationships:
+.. _dotnet-client-relationships:
```
Comment (cbush) [inline source/sdk/android/fundamentals/relationships.txt:1]: I think you got this backwards. The Android one is in the /dotnet/ directory and the .NET one is in the /android/ directory?

**Review by cbush — CHANGES_REQUESTED** [review]
Thanks for doing this! Quick glance - early issue spotted. Otherwise looking good!

File: source/sdk/android/fundamentals/relationships.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/sdk/android/fundamentals/relationships.txt:1]: Ha, thank you -- I *knew* this would happen with at least one of the pages.

Comment (nathan-contino-mongo) [reply source/sdk/android/fundamentals/relationships.txt]: Just did a final search for java/android/kt mentions in the sdk/dotnet directory, and it looks good except for *one* mention of deep linking, which is relevant to cross-platform behavior. Let me know if you see any more!

**Review by cbush — APPROVED** [review]

**Pull request merged**